crates/codec/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+//! Special purpose binary encoding format.
```

Comment:
This could also go into the `Cargo.toml`'s [`description`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-description-field) field.